Mergin/utils.py

```diff
@@ -99,3 +99,11 @@ def changes_from_metadata(metadata):
     updated = ", ".join(f['path'] for f in metadata['updated'])
     renamed = ", ".join(f"{f['path']} -> {f['new_path']}" for f in metadata['renamed'])
     return added, removed, updated, renamed
+
+
+def get_plugin_version():
+    with open(os.path.join(os.path.dirname(__file__),"metadata.txt"), 'r') as f:
+        for line in f:
```

Comment:
using python config parser seems more elegant to me https://docs.python.org/3.6/library/configparser.html